.github/workflows/test.yml

```diff
@@ -29,9 +27,8 @@ jobs:
   test-java:
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
one more thing - do you foresee slow builds for this repo?  Martin repo for example uses about 1.5hrs for each build, and we have 10-20 a day.

Reply:
I do foresee potentially running benchmarks in the future, which could take more time. But not 1.5hrs, no! Here I'm trying to be conservative with resources since I'm assuming that the limit of 2000 minutes is across all maplibre org repos and I don't have a sense of the total minutes others take.

Reply:
In that case I would rather keep it -- `macOS` is a major pain when it comes to Linux compatibility - i.e. lots of shell scripts break because some flags have different meaning.  On the other hand, having macOS target ensures all of our contributors can be certain their code works on every platform - and macOS people won't have surprises if they start contributing.

As for benchmarks - we can easily run them conditionally on linux only when we have them.

Reply:
Okay, sounds good 👍🏼

Reply:
Added back in f6ed6ae. BTW, I normally develop using macOS so I'm also a big fan of keeping this working :) Good to hear you are not worried about the resource use. Easy to revisit if it becomes an issue.

Reply:
OMG, you use Mac? jk :) 
 
Developers and their time and on-boarding experience are far more valuable than a few extra minutes of a `macOS` runner ;)